bin/pgenv

```diff
@@ -636,6 +866,58 @@ EOF
         exit
         ;;
 
+    config|configuration)
+        action=$2
+        v=$3
+
+        # which version does the user wants?
+        case "$v" in
+            version|current)
```

Comment:
I find the use of the term "version" here confusing. I think it should just be "current".

Reply:
`version` is to remind the user of the `version` command (I mean `pgenv version`), so that she can get habit to use `version` to know it points to the current instance and the current configuration.
The fact is that I believe `current` is a much more pleasant term than `version` in both `pgenv config current` and `pgenv current` instead of `pgenv version`.

Reply:
Hrm. I was following rbenv and plenv on this, but have no particular objection to changing it.